src/DataTransferObject.php

```diff
@@ -82,10 +82,40 @@ public function except(string ...$keys): DataTransferObject
     public function toArray(): array
     {
         if (count($this->onlyKeys)) {
-            return Arr::only($this->all(), $this->onlyKeys);
+            $array = Arr::only($this->all(), $this->onlyKeys);
```

Comment:
We can combine this if/else check into one:

```php
$array = Arr::only(
    Arr:except($this->all, $this->exceptKeys), 
    $this->onlyKeys
);
```

I wanted to refactor this for a while now, but you might as well do it in this PR.

Reply:
Ok

Reply:
Hm...it's a breaking change. By default $onlyKeys property is a []. Arr::only with empty second parameter always will be return empty array.

I suggest this option:

```php
$array = Arr::except(
     count($this->onlyKeys) ? Arr::only($this->all(), $this->onlyKeys) : $this->all(),
     $this->exceptKeys
);
```

Reply:
Ah, too bad. Let's than keep it the way you implemented it for now. I find that to be clearer than the shorthand approach.

I'll probably merge this later today! 👍